### ISSUE TITLE
Make columns like Notion's columns

### ISSUE THREAD

**Issue by ghost** [toeverything/AFFiNE#7479]
### Description

I'm new in the app, i loved the app! it's really good, i suffered to find an app that it's a Notion open-source alternative, and Affine was the perfect sollution for me.

I have a feature idea for the app, can you guys make a column-like display, like what the Notion has, with 2, 3, 4 and 5 columns. I like this feature from Notion, but i want to do a collaborative project with a friend and Notion it's very limited and we can't afford the payment plans. 

We would like to put an image aside of a text, like a lot of websites do.  Me and my friend would appreciate a lot if that feature was added in the app!

### Anything else?

![image](https://github.com/user-attachments/assets/121940d7-2086-448a-85b4-2b73eb89e945)
This is an example that my friend do in his private Notion workspace

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by xmflswood** [comment]
I'm looking forward to having this feature.

**Comment by marcopadillag** [comment]
It would be a really good feature.

**Comment by Jarnooow** [comment]
This is the only reason why I can't switch yet. I use columns a lot. Would be amazing if it could be added.

**Comment by AndreasWidmer** [comment]
+1 really looking for this!

**Comment by walero-moon** [comment]
+1

**Comment by icarus-00** [comment]
i was about to write an issue for this, especially that there're no inline images, which annoys me somewhat, but having multi-column support would hopefully fix that

**Comment by icarus-00** [comment]
i'd put that as a feature request, this is something that we hopefully could get before 0.20

**Comment by Woisol** [comment]
Still cant switch now on 02-28. is there any update?

**Comment by hisuperwen** [comment]
+1

**Comment by zag13** [comment]
+1

**Comment by apertso** [comment]
+1

**Comment by Magasul** [comment]
+1

**Comment by blacklightpy** [comment]
+1, one of the reasons I'm not switching yet. I like my Notion Dashboards that utilize the space well.

**Comment by YoshoTosho** [comment]
+1 . One of the reasons why I'm not ready yet to switch. Most pages I have are well divided into columns, and it doesn't fit at all with AFFiNE.

**Comment by justiceserv** [comment]
+1

**Comment by paufer** [comment]
+1

**Comment by Pascal-Flores** [comment]
+1

**Comment by lovelycbm** [comment]
+1

**Comment by thompsonpat** [comment]
The only thing preventing me from switching over. Columns are too important to go without.

**Comment by bernie43** [comment]
You can actually kind of recreate this with Edgeless Canvas already.
Your page will be the "main column", and then you can create several notes on the canvas, and rearrange them so they form the other columns.
I made an example with some dummy data:

<img width="350" alt="Image" src="https://github.com/user-attachments/assets/17abbdf3-5675-4cc9-9632-9b4ac60964cc" />

**Comment by icarus-00** [comment]
it is mostly just a workaround, having to switch between modes and views, and use mouse to draw and organize, i do that to align images with text in either side, but having a multi-column layout without switching would be nice especially for those who use the Page mode most of the time, but i don't how would inserted groups or frames from edgeless would behave with multi-column

**Comment by rescoldo** [comment]
+1000

**Comment by lucabockmann** [comment]
+1

**Comment by mogita** [comment]
Yes, column is so nice to have. It's the era of wide screens, we should use the space with efficiency.

**Comment by mwhouser** [comment]
+1 please

**Comment by mesto-dev** [comment]
+1 please, so important and usefull... why not ? It is necessary to be able to make a functional, clear and well organized pages

**Comment by violetbeam** [comment]
+1

**Comment by chrisjensen** [comment]
+1

**Comment by nothing2obvi** [comment]
+1

**Comment by steveoriol** [comment]
+1

**Comment by jmbillard** [comment]
+1

**Comment by phucnht** [comment]
Any updates? Could we have a first version of the feature for the next release?

**Comment by mrvinrsk** [comment]
The only reason as of now why I can't switch. Even though I generally like AFFiNE better due to e.g. the collapsable Info on database pages, but I need Columns.

+1

**Comment by Jonathan-R23** [comment]
Thank you for the great work on AFFiNE — it’s an amazing project and I really appreciate the effort from the team.

It would be great to have the ability to organize content using rows inside documents. This could really improve layout flexibility and help structure information more clearly.

**Comment by blacklightpy** [comment]
This is literally the most commented on issue here and has being going on for 2 years, but there hasn't been any response from the team..

**Comment by NickvonJ** [comment]
+1

**Comment by JPBeckner** [comment]
+1

**Comment by rescoldo** [comment]
It would be great if they listened to their users and implemented something as basic as this. While Notion has AI agents and MCP connectivity with Claude, Affine still doesn't have two columns to better organize information... 👎

**Comment by vsgusev** [comment]
+1

**Comment by AvirupDas** [comment]
+1

**Comment by JxstMichie** [comment]
+1

**Comment by GoblinGanzo1999** [comment]
Hey AFFiNE team, an update on this would be much appreciated!

**Comment by S-A-Adit** [comment]
@darkskygit I would like to work on this issue if it is still open.
Thank You.
